docs/tools/android_lint.md

```diff
@@ -0,0 +1,9 @@
+# Android Lint
+[Android Lint](https://developer.android.com/studio/write/lint.html) is a linter and static analysis tool for Android projects which can detect bugs
+and potential issues in code, resources and configuration files.
+
+Be aware that Lint just supports Kotlin since version 3.0.1 of the [Android Gradle Plugin](https://developer.android.com/studio/releases/gradle-plugin.html).
```

Comment:
❌ it's 3.1.0, not 3.0.1

I would rephrase for consistency with the other tools: `Please note that Android Lint only supports Kotlin projects from the command line since version 3.1.0 of the ...`